resources/tentative/complex-static-html/src/big-dom-generator/gen-css.js

```diff
@@ -0,0 +1,231 @@
+import { LCG } from 'random-seedable';
+import { SEED, MAX_SELECTOR_LEN } from './params.js';
+
+const rando = new LCG(SEED);
+
+// The generator assumes the page has the following structure,
+// and it needs to be updated if the structure changes.
+// TODO: make it less hard-coded.
+/* 
+  <body class="ui">
+    <div class="ui main-ui">
+      <div class="ui show-more">
+      <div class="ui top-bar">
+      <div class="ui ribbon">
+      <div class="ui tree-area">
+      <div class="ui todo-area">
+        <div class="absolutely-positioned-element">
+          <section class="todoapp">
+            <header class="header">
+            <main class="main">
+              <input class="toggle-all-container">
+              <ul class="todo-list">
+                <li class="li-0-0">
+                  <div class="view-0">
+                <li class="li-1-0">
+                  <div class="view-0">
+                ...
+              </ul>
+            </main>
+          </section>
+        </div>
+      </div>
+    </div>
+</body>
+*/
+
+// Returns a classname for the element at depth `depth` and in position `index`.
+const getClassname = (depth, index) => {
+  switch (depth) {
+    case 7:
+      return `.view-${index}`;
+    case 6:
+      return `.li-${index}-0`;
+    case 5:
+      if (index === 0) {
```

Comment:
Nit: These should be indented by 4 spaces, not 2 spaces,  and there should be no curly braces around single line statements.

Reply:
Done.